libretroshare/src/serialiser/rstypeserializer.h

```diff
@@ -909,6 +910,27 @@ struct RsTypeSerializer
 	                       t_RsTlvList<TLV_CLASS,TLV_TYPE>& member,
 	                       RsJson& jDoc );
 
+	[[noreturn]] static void fatalUnknownSerialJob(int j)
```

Comment:
what's this????

Reply:
https://en.cppreference.com/w/cpp/language/attributes/noreturn  it tells that the function won't return, aka if the program get there it will be terminated like when you call exit(...)